CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/OpenGraph.java

```diff
@@ -113,6 +116,10 @@ public void callService(final PluginParameters parameters, final InputStream in,
             mapper.writeValue(out, root);
         } catch (final GraphParseException | FileNotFoundException ex) {
             throw new RestServiceException(HTTP_UNPROCESSABLE_ENTITY, ex.getMessage());
+        } catch (final InterruptedException ex) {
+            Thread.currentThread().interrupt();
```

Comment:
Similar to `NewGraph` are you able to add either a log entry or throw an exception here?